tests/src/test/scala/com/nvidia/spark/rapids/GpuPartitioningSuite.scala

```diff
@@ -42,8 +42,18 @@ class GpuPartitioningSuite extends FunSuite with Arm {
     }
   }
 
+  private def extractBases(batch: ColumnarBatch): Array[ColumnVector] = {
+    if (GpuPackedTableColumn.isBatchPacked(batch)) {
+      val packedColumn = batch.column(0).asInstanceOf[GpuPackedTableColumn]
+      val table = packedColumn.getContiguousTable.getTable
```

Comment:
nit: This feels a little off.  We just grabbed the table and pulled out the columns. It might be good to have a comment here about how packedColumn still owns the table and the reference counts have not changed for it.